src/java.base/share/classes/module-info.java

```diff
@@ -169,6 +169,7 @@
         java.management,
         java.rmi,
         jdk.charsets,
+        jdk.internal.le,
```

Comment:
What code in JLine is using shared secrets? I wonder what we need to change to avoid this.

Reply:
Oops, sorry. I forgot to remove these (qualified export of `jdk.internal.access` and `sun.nio.cs`) after a refactoring. Will remove them in a next update.